### DISCUSSION TITLE
Random type of class - Magic or Might

### DISCUSSION THREAD

**Discussion by denierlexiese** [ihhub/fheroes2#8890] [Ideas]
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.

### Describe the problem requiring a solution

Might be interesting on the race selection to be able to select 'Random Might' or 'Random 'Magic' hero, instead of just Random.

### Describe the possible solution

Might be interesting on the race selection to be able to select 'Random Might' or 'Random 'Magic' hero, instead of just Random.

**Comment by ihhub** [comment]
Moving this to discussion section. This is not a straightforward idea to accept and implement.
